src/test/java/lotto/domain/LottoTest.java

```diff
@@ -0,0 +1,16 @@
+package lotto.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class LottoTest {
+
+    @Test
+    void 동일한번호_생성시_예외() {
+        Assertions.assertThatIllegalArgumentException()
+                .isThrownBy(() -> new Lotto(new Integer[]{1, 2, 3, 3, 4, 5}));
```

Comment:
```suggestion
                .isThrownBy(() -> new Lotto(1, 2, 3, 3, 4, 5));
```
위와 같이 구현 가능하도록 가변 인자를 받을 수 있는 생성자를 추가하면 어떨까?